packages/inquirer/test/specs/prompts/confirm.test.js

```diff
@@ -85,9 +95,35 @@ describe('`confirm` prompt', () => {
       rl.emit('line', 'No');
     }));
 
-  it('should parse every other string value to boolean false', () =>
+  it('should parse every other string value to default (unset)', () =>
     new Promise((done) => {
       confirm.run().then((answer) => {
+        expect(answer).toEqual(true);
+        done();
+      });
+
+      rl.emit('line', 'bla bla foo');
+    }));
+
+  it('should parse every other string value to default (true)', () =>
+    new Promise((done) => {
+      fixture.default = true;
+      const trueConfirm = new Confirm(fixture, rl);
+
+      trueConfirm.run().then((answer) => {
+        expect(answer).toEqual(true);
+        done();
+      });
+
+      rl.emit('line', 'bla bla foo');
+    }));
```

Comment:
First, I'm happy to merge as-is since those are very old tests. But just FYI, that pattern you see in the test files is just a codemod to move to vitest; not a preference haha.

You could remove the Promise wrapper in favor of

```suggestion
  it('should parse every other string value to default (true)', async () => {
    fixture.default = true;
    const trueConfirm = new Confirm(fixture, rl);

    const answer = trueConfirm.run();

    rl.emit('line', 'bla bla foo');
    await expect(answer).resolves.toEqual(true);
  });
```

Reply:
OK. I will change them to async/await in another PR.